source/transactions/tests/unified/do-not-retry-read-in-transaction.json

```diff
@@ -0,0 +1,116 @@
+{
+  "description": "do not retry read in a transaction",
+  "schemaVersion": "1.4",
+  "runOnRequirements": [
+    {
+      "minServerVersion": "4.0.0",
+      "topologies": [
+        "single",
```

Comment:
I errantly added this topology, and didn't wait for tests to complete on Evergreen.  Will remove in subsequent commit.

Reply:
https://github.com/mongodb/specifications/commit/bafb024be8202b1676a3c6632e8a3eb9cf36209b